light_node/src/main.rs

```diff
@@ -364,6 +364,7 @@ fn block_on_actors(
             shell_compatibility_version,
             env.p2p,
             env.identity.expected_pow,
+            Arc::new(init_storage_data.chain_id),
```

Comment:
Why `Arc` ? Do we need chain_id in `PeerManager`?

Reply:
@tizoc 
arc removed, chain_id is needed in tezedge_state